modules/payment_reference/src/Plugin/Field/FieldType/PaymentReference.php

```diff
@@ -38,7 +38,6 @@ class PaymentReference extends ConfigurableEntityReferenceItem {
    */
   public static function defaultStorageSettings() {
     return [
-      'target_bundle' => 'payment_reference',
```

Comment:
@Berdir, a new pull request is here: https://github.com/bartfeenstra/drupal-payment/pull/7